src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java

```diff
@@ -0,0 +1,79 @@
+package vaishnavi_SeleniumBasics;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import jdk.management.resource.internal.inst.SocketOutputStreamRMHooks;
+
+public class VerifyFirstWebPage {
+
+	public static void main(String[] args) {
+		System.setProperty("webdriver.chrome.driver", ".\\resources\\windows//chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
+		driver.get("file:///C:/Users/vaish/Desktop/Selenium_Recordings/TechnoCredits.html");
+
+		// Trying to verify title of the page
+		if (driver.getTitle().equals("TECHNOCREDITS"))
+			System.out.println("Yes, You are on a correct Page with title TECHNOCREDITS");
+		else
+			System.out.println("You are on a page of title: " + driver.getTitle());
+
+		// Filling all the details of form
+		driver.findElement(By.id("first name")).sendKeys("Vaishnavi");
+		driver.findElement(By.id("last name")).sendKeys("Vaidya");
+		driver.findElement(By.id("E-mail")).sendKeys("Vaishnavivaidya06@gmail.com");
+		driver.findElement(By.id("Company Name")).sendKeys("Atos-Syntel");
+
+		WebElement maleRadioButton = driver.findElement(By.id("male"));
+		maleRadioButton.click();
+
+		WebElement experience = driver.findElement(By.id("1-3"));
```

Comment:
You should only store any value in webElement when you want to perform more than 1 action. 
But here you are only performing one action which click() so you should write like this:
 driver.findElement(By.id("1-3")).click();

Reply:
-- Actually I am using this to verify the reset condition later as well , Please let me know if still I should remove this.

Reply:
No dear.